From 39e4672d86ce82075d97af98f0a13ea41c83435e Mon Sep 17 00:00:00 2001
From: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
Date: Fri, 8 Jan 2021 16:37:28 -0800
Subject: [PATCH] Add Requires to xmlsec1

a.) Add Require of lgnutls on xmlsec1-gnutls
b.) Add Require of lgcrypt, lgpg-error on xmlsec1-gcrypt
c.) Add Require of lssl, lcrypto on xmlsec1-openssl

Signed-off-by: Karthik Prabhu Vinod <karthik.prabhu.vinod@intel.com>
---
 xmlsec-gcrypt.pc.in  | 2 +-
 xmlsec-gnutls.pc.in  | 2 +-
 xmlsec-openssl.pc.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/xmlsec-gcrypt.pc.in b/xmlsec-gcrypt.pc.in
index 79f8ce4..f20ddae 100644
--- a/xmlsec-gcrypt.pc.in
+++ b/xmlsec-gcrypt.pc.in
@@ -6,7 +6,7 @@ includedir=@includedir@
 Name: xmlsec1-gcrypt
 Version: @VERSION@
 Description: XML Security Library implements XML Signature and XML Encryption standards
-Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@
+Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@ libgcrypt gpg-error
 Cflags: @XMLSEC_GCRYPT_CFLAGS@
 Cflags.private: -DXMLSEC_STATIC
 Libs: @XMLSEC_GCRYPT_LIBS@
diff --git a/xmlsec-gnutls.pc.in b/xmlsec-gnutls.pc.in
index 712e1a4..fe00f5b 100644
--- a/xmlsec-gnutls.pc.in
+++ b/xmlsec-gnutls.pc.in
@@ -6,7 +6,7 @@ includedir=@includedir@
 Name: xmlsec1-gnutls
 Version: @VERSION@
 Description: XML Security Library implements XML Signature and XML Encryption standards
-Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@
+Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@ gnutls
 Cflags: @XMLSEC_GNUTLS_CFLAGS@
 Cflags.private: -DXMLSEC_STATIC
 Libs: @XMLSEC_GNUTLS_LIBS@
diff --git a/xmlsec-openssl.pc.in b/xmlsec-openssl.pc.in
index af3ae29..b403002 100644
--- a/xmlsec-openssl.pc.in
+++ b/xmlsec-openssl.pc.in
@@ -6,7 +6,7 @@ includedir=@includedir@
 Name: xmlsec1-openssl
 Version: @VERSION@
 Description: XML Security Library implements XML Signature and XML Encryption standards
-Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@
+Requires: libxml-2.0 >= @LIBXML_MIN_VERSION@ @LIBXSLT_PC_FILE_COND@ libssl libcrypto
 Cflags: @XMLSEC_OPENSSL_CFLAGS@
 Cflags.private: -DXMLSEC_STATIC
 Libs: @XMLSEC_OPENSSL_LIBS@
-- 
2.30.0

